### PR TITLE
Replace tdex with dex vulpem

### LINKED ISSUE #38: Homepage: Remove TDEX-CLI section for dex.vulpem.com
Remove TDEX-CLI section, in favor of dex.vulpem web app.

Taking a screenshot is more than enough.

### PR DESCRIPTION
**Pull request by bordalix**
Fixes #38 

This PR changes `TDex Cli` with `TDEX`:
- On homepage, with screenshot of dex.vulpem.com
- On footer and menus
- On CSS, class `tdex-cli` with `tdex`
- Adds new image `dex.mockup.png`

@tiero please review

### CODE REVIEW

File: src/401.html [hunk 1]
```diff
@@ -65,12 +65,11 @@
...
             </a>
-            <a href="https://github.com/vulpemventures/tdex-cli" target="_blank"
-              class="nav-dropdown-link w-inline-block"><img src="images/Tdex_logo_soloT.svg" loading="lazy" alt=""
-                class="nav-dropdown-image">
+            <a href="https://dex.vulpem.com" target="_blank" class="nav-dropdown-link w-inline-block">
+              <img src="images/Tdex_logo_soloT.svg" loading="lazy" alt="" class="nav-dropdown-image">
               <div class="content">
-                <div class="dropdown-nav-item-title"><strong class="text-regular">TDex CLI</strong></div>
-                <div class="paragraph">Non-custodial CLI for making trades on TDex Network via atomic swaps. </div>
+                <div class="dropdown-nav-item-title"><strong class="text-regular">Dex Vulpem</strong></div>
```
Comment (tiero) [inline src/401.html]:
```suggestion
                <div class="dropdown-nav-item-title"><strong class="text-regular">TDEX</strong></div>
```

File: src/404.html [hunk 2]
```diff
@@ -57,10 +57,11 @@
                   <div class="paragraph">Browser-based tool for Liquid developers and advanced users</div>
                 </div>
               </a>
-              <a href="https://github.com/vulpemventures/tdex-cli" target="_blank" class="nav-dropdown-link w-inline-block"><img src="images/Tdex_logo_soloT.svg" loading="lazy" alt="" class="nav-dropdown-image">
+              <a href="https://dex.vulpem.com" target="_blank" class="nav-dropdown-link w-inline-block">
+                <img src="images/Tdex_logo_soloT.svg" loading="lazy" alt="" class="nav-dropdown-image">
                 <div class="content">
-                  <div class="dropdown-nav-item-title"><strong class="text-regular">TDex CLI</strong></div>
-                  <div class="paragraph">Non-custodial CLI for making trades on TDex Network via atomic swaps. </div>
+                  <div class="dropdown-nav-item-title"><strong class="text-regular">Dex Vulpem</strong></div>
```
Comment (tiero) [inline src/404.html]:
```suggestion
                  <div class="dropdown-nav-item-title"><strong class="text-regular">TDEX</strong></div>
```

File: src/about.html [hunk 3]
```diff
@@ -65,12 +65,11 @@
...
             </a>
-            <a href="https://github.com/vulpemventures/tdex-cli" target="_blank"
-              class="nav-dropdown-link w-inline-block"><img src="images/Tdex_logo_soloT.svg" loading="lazy" alt=""
-                class="nav-dropdown-image">
+            <a href="https://dex.vulpem.com" target="_blank" class="nav-dropdown-link w-inline-block">
+              <img src="images/Tdex_logo_soloT.svg" loading="lazy" alt="" class="nav-dropdown-image">
               <div class="content">
-                <div class="dropdown-nav-item-title"><strong class="text-regular">TDex CLI</strong></div>
-                <div class="paragraph">Non-custodial CLI for making trades on TDex Network via atomic swaps. </div>
+                <div class="dropdown-nav-item-title"><strong class="text-regular">Dex Vulpem</strong></div>
```
Comment (tiero) [inline src/about.html]:
```suggestion
                <div class="dropdown-nav-item-title"><strong class="text-regular">TDEX</strong></div>
```

File: src/carrers.html [hunk 4]
```diff
@@ -65,12 +65,11 @@
...
             </a>
-            <a href="https://github.com/vulpemventures/tdex-cli" target="_blank"
-              class="nav-dropdown-link w-inline-block"><img src="images/Tdex_logo_soloT.svg" loading="lazy" alt=""
-                class="nav-dropdown-image">
+            <a href="https://dex.vulpem.com" target="_blank" class="nav-dropdown-link w-inline-block">
+              <img src="images/Tdex_logo_soloT.svg" loading="lazy" alt="" class="nav-dropdown-image">
               <div class="content">
-                <div class="dropdown-nav-item-title"><strong class="text-regular">TDex CLI</strong></div>
-                <div class="paragraph">Non-custodial CLI for making trades on TDex Network via atomic swaps. </div>
+                <div class="dropdown-nav-item-title"><strong class="text-regular">Dex Vulpem</strong></div>
```
Comment (tiero) [inline src/carrers.html]:
```suggestion
                <div class="dropdown-nav-item-title"><strong class="text-regular">TDEX</strong></div>
```

File: src/index.html [hunk 5]
```diff
@@ -65,12 +65,11 @@
...
             </a>
-            <a href="https://github.com/vulpemventures/tdex-cli" target="_blank"
-              class="nav-dropdown-link w-inline-block"><img src="images/Tdex_logo_soloT.svg" loading="lazy" alt=""
-                class="nav-dropdown-image">
+            <a href="https://dex.vulpem.com" target="_blank" class="nav-dropdown-link w-inline-block">
+              <img src="images/Tdex_logo_soloT.svg" loading="lazy" alt="" class="nav-dropdown-image">
               <div class="content">
-                <div class="dropdown-nav-item-title"><strong class="text-regular">TDex CLI</strong></div>
-                <div class="paragraph">Non-custodial CLI for making trades on TDex Network via atomic swaps. </div>
+                <div class="dropdown-nav-item-title"><strong class="text-regular">Dex Vulpem</strong></div>
```
Comment (tiero) [inline src/index.html]:
```suggestion
                <div class="dropdown-nav-item-title"><strong class="text-regular">TDEX</strong></div>
```

File: src/marina.html [hunk 6]
```diff
@@ -69,12 +69,11 @@
...
             </a>
-            <a href="https://github.com/vulpemventures/tdex-cli" target="_blank"
-              class="nav-dropdown-link w-inline-block"><img src="images/Tdex_logo_soloT.svg" loading="lazy" alt=""
-                class="nav-dropdown-image">
+            <a href="https://dex.vulpem.com" target="_blank" class="nav-dropdown-link w-inline-block">
+              <img src="images/Tdex_logo_soloT.svg" loading="lazy" alt="" class="nav-dropdown-image">
               <div class="content">
-                <div class="dropdown-nav-item-title"><strong class="text-regular">TDex CLI</strong></div>
-                <div class="paragraph">Non-custodial CLI for making trades on TDex Network via atomic swaps. </div>
+                <div class="dropdown-nav-item-title"><strong class="text-regular">Dex Vulpem</strong></div>
```
Comment (tiero) [inline src/marina.html]:
```suggestion
                <div class="dropdown-nav-item-title"><strong class="text-regular">TDEX</strong></div>
```

File: src/nigiri.html [hunk 7]
```diff
@@ -71,12 +71,11 @@
...
             </a>
-            <a href="https://github.com/vulpemventures/tdex-cli" target="_blank"
-              class="nav-dropdown-link w-inline-block"><img src="images/Tdex_logo_soloT.svg" loading="lazy" alt=""
-                class="nav-dropdown-image">
+            <a href="https://dex.vulpem.com" target="_blank" class="nav-dropdown-link w-inline-block">
+              <img src="images/Tdex_logo_soloT.svg" loading="lazy" alt="" class="nav-dropdown-image">
               <div class="content">
-                <div class="dropdown-nav-item-title"><strong class="text-regular">TDex CLI</strong></div>
-                <div class="paragraph">Non-custodial CLI for making trades on TDex Network via atomic swaps. </div>
+                <div class="dropdown-nav-item-title"><strong class="text-regular">Dex Vulpem</strong></div>
```
Comment (tiero) [inline src/nigiri.html]:
```suggestion
                <div class="dropdown-nav-item-title"><strong class="text-regular">TDEX</strong></div>
```

File: src/solutions.html [hunk 8]
```diff
@@ -65,12 +65,11 @@
...
             </a>
-            <a href="https://github.com/vulpemventures/tdex-cli" target="_blank"
-              class="nav-dropdown-link w-inline-block"><img src="images/Tdex_logo_soloT.svg" loading="lazy" alt=""
-                class="nav-dropdown-image">
+            <a href="https://dex.vulpem.com" target="_blank" class="nav-dropdown-link w-inline-block">
+              <img src="images/Tdex_logo_soloT.svg" loading="lazy" alt="" class="nav-dropdown-image">
               <div class="content">
-                <div class="dropdown-nav-item-title"><strong class="text-regular">TDex CLI</strong></div>
-                <div class="paragraph">Non-custodial CLI for making trades on TDex Network via atomic swaps. </div>
+                <div class="dropdown-nav-item-title"><strong class="text-regular">Dex Vulpem</strong></div>
```
Comment (tiero) [inline src/solutions.html]:
```suggestion
                <div class="dropdown-nav-item-title"><strong class="text-regular">TDEX</strong></div>
```

**Review by tiero — CHANGES_REQUESTED** [review]
all TDEX links should have target="_blank"

**Comment by bordalix** [comment]
> all TDEX links should have target="_blank"

Fixed in 27b3c24

**Pull request merged**